modules/monitoring-accessing-metrics-of-all-services.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="accessing-metrics-of-your-service_{context}"]
+= Accessing metrics of all services
+
+If you are an administrator or a user with view permissions for all namespaces, you can access metrics of all services together in a single view.
+
+.Prerequisites
+
+* Have the `oc` command installed.
+* Log in as a cluster administrator or a user with view permissions for all namespaces.
+
+.Procedure
+
+. List the URL for the Thanos Querier UI:
++
+----
+$ oc -n openshift-monitoring get routes
+NAME                HOST/PORT
+...
+thanos-querier      thanos-querier-openshift-monitoring.apps.example.devcluster.openshift.com
+----
+
+. Navigate to the listed URL. Here you can see user alerting rules and the respective alerts from all namespaces.
```

Comment:
The Thanos Querier UI doesn't display alerts AFAIK. Cc @s-urbaniak for confirmation.

Reply:
Correct, alerts are being displayed by Thanos Ruler.

Reply:
But Thanos Ruler still does not display all alerts. It only shows those which are deployed there, there could be others deployed on the user workload prometheus. In 4.5, except for listing all Prometheus Rules using `oc get -n <your_namespace> get prometheusrule` there is no good way of listing those.